ros_buildfarm/templates/snippet/rosdep_init.Dockerfile.em

```diff
@@ -0,0 +1,8 @@
+RUN mkdir -p /etc/ros/rosdep/sources.list.d
+
+@[for url in custom_rosdep_urls]@
+RUN (cd /etc/ros/rosdep/sources.list.d/ && wget --progress=dot @url )
+@[end for]@
+@[if not custom_rosdep_urls]@
+RUN rosdep init
```

Comment:
I would suggest running rosdep init either way. And not override the default sources, but add another file side by side with the additional sources. You should be able to set precidence using the number at the front of the file.